swan-lake/featured-use-cases/manage-data-persistence-with-bal-persist.md

```diff
@@ -0,0 +1,97 @@
+
+redirect_from:
+- /learn/manage-data-persistence-with-bal-persist/
+- /learn/manage-data-persistence-with-bal-persist
+- /learn/manage-data-persistence-with-bal-persist
+- /learn/getting-started/manage-data-persistence-with-bal-persist/
+- /learn/getting-started/manage_data_persistence-with-bal-persist
+---
+
+## Set up the prerequisites
+
+To complete this tutorial, you need:
+
+1. [Ballerina 2201.6.0 (Swan Lake)](/learn/install-ballerina/set-up-ballerina/) or greater
+2. A text editor
+>**Tip:** Preferably, <a href="https://code.visualstudio.com/" target="_blank">Visual Studio Code</a> with the
+<a href="https://wso2.com/ballerina/vscode/docs/get-started/install-the-extension/" target="_blank">Ballerina extension</a> installed.
+3. A command terminal
+
+## Understand the implementation
+
+This tutorial describes how to interact with the data store and perform operations against it using Ballerina Persistence via a basic use case of creating, reading, updating, and deleting records on a data store in an organization. It also elaborates on how you can create an HTTP RESTful API using Ballerina that can be used to perform basic CRUD operations on the data store.
+
+![Data Service Architecture](/learn/images/data-service-architecture.png "Data Service Architecture")
+
+>**Info:** In this tutorial, we use an in-memory data store to make it simple. The outlined methodology can be used to work with MySQL and Google Sheets data stores as the Bal persist currently supports three data stores In-memory tables, MySQL database, and Google Sheets.
+
+## Create a Ballerina package
+
+Ballerina uses packages to group code. You need to create a Ballerina package and write the business logic in it. In the terminal, execute the command below to create the Ballerina package for the implementation.
+
+> **Info:** For more information on Ballerina packages, see [Organize Ballerina code](/learn/organize-ballerina-code/).
+
+```
+$ bal new rainier
+``` 
+
+This creates a directory named `rainier` with the files below.
+
+```
+.
+├── rainier
+│   ├── Ballerina.toml
+│   └── main.bal
+```
+
+## Initialize Bal persist in the project
+
+The Ballerina Persistence initialization takes care of the basics needed to start the development. In the terminal, execute the command below to initialize persistence in the Ballerina package.
+
+```
+$ bal persist init --module store
+```
+
+Along with the `bal persist init` command, you can specify the `datastore` that you are going to use in the application and also the `module` that you need to add the generated code to. Both these parameters are optional. If you don’t provide them, the datastore will be set to `in-memory` and the module will be set to `default` module by default.
+
+This changes the Ballerina package like below,
```

Comment:
Instead of `like below` use `as follows`?

`This changes the Ballerina package as follows.`